pkg/workload/schemachange/schemachange.go

```diff
@@ -248,7 +248,7 @@ func (s *schemaChange) Ops(
 			seqNum:             seqNum,
 			errorRate:          s.errorRate,
 			enumPct:            s.enumPct,
-			rng:                rng,
+			rng:                randutil.NewTestRandWithSeed(seed),
```

Comment:
Do we want to set a derived seed (e.g., `seed+i`) for each worker so that they perform different operations?